server/main.go

```diff
@@ -89,7 +89,7 @@ func handleConn(w http.ResponseWriter, r *http.Request) {
 
 	// Generate the UUID and the site ID for the client.
 	clientID := uuid.New()
-	siteID := strconv.Itoa(len(activeClients))
+	siteID := strconv.Itoa(len(activeClients)) //TODO: change siteIDs to be a monotonically increasing global variable
```

Comment:
This seems plausible; could we move this to a global variable instead?

We could do an increment operation here.

Reply:
Yeah, I think moving siteID to a global variable and incrementing it here every time a new client joins is the correct way to do it. The way it is now will lead to collisions if one client leaves and another client joins.

Reply:
Could we do this in this PR itself?

Reply:
Resolved in https://github.com/burntcarrot/rowix/pull/22/commits/298bcc8a6e5b8aa3da0673f6a7a43296289e533f.